src/azure-cli/azure/cli/command_modules/network/_params.py

```diff
@@ -1255,6 +1255,11 @@ def load_arguments(self, _):
         c.argument('virtual_network', virtual_network_name_type, options_list='--vnet', help="Name or ID of a virtual network that contains a subnet named 'GatewaySubnet'.")
         c.extra('address_prefixes', options_list='--address-prefixes', help='List of address prefixes for the VPN gateway.  Prerequisite for uploading certificates.', nargs='+')
 
+    with self.argument_context('network vnet-gateway aad assign', min_api='2019-04-01') as c:
+        c.argument('aad_tenant', help='The AADTenant URI of the VirtualNetworkGateway.')
```

Comment:
besides, since these three arguments have been under aad group command, do we need `aad_`?

Reply:
Sounds good. Fix it.